src/sentry/snuba/discover.py

```diff
@@ -1116,7 +1115,7 @@ def find_histogram_min_max(fields, min_value, max_value, user_query, params, dat
 
     if min_value is None:
         min_values = [row[get_function_alias(column)] for column in min_columns]
-        min_values = list(filter(lambda v: v is not None, min_values))
+        min_values = list(list(filter(lambda v: v is not None, min_values)))
```

Comment:
This seems really unnecessary.

Reply:
This is done in an optimization pass that I linked in the description. I guess we'd prefer it to be all rolled up in 1 PR though. https://github.com/getsentry/sentry/pull/28080#issuecomment-908578834